README.md

```diff
@@ -1,29 +1,41 @@
 mstpd (Multiple Spanning Tree Protocol daemon)
 ==========
 
-Important Note
-----------
-
-As the note (near the title says) this is not the original repo of **mstpd**.
-
-It's a "fork" from SourceForge and the origin link is:
- https://sourceforge.net/projects/mstpd/
-
-And the original author is: Vitalii Demianets <dvitasgs@gmail.com>
-
-Description
------------
-
-**msptd** implements the IEEE 802.1s Multiple Spanning Tree Protocol.
+MSTPD is an open source user-space daemon licensed under GPLv2.
+At the moment the latest version 0.04 is reported to be compliant with IXIA ANVL RSTP test suite, with the notable exception of looped-back BPDUs (see discussion on the matter on the Implementation Features wiki page: https://github.com/mstpd/mstpd/wiki/ImplementationFeatures).
```

Comment:
We don't have a 0.04 Git tag yet. Nor is 0.03 tagged. Should we drop the version number here?